zuul.d/content_provider.yaml

```diff
@@ -2,3 +2,24 @@
 - job:
     name: content-provider-base
     parent: openstack-k8s-operators-content-provider
+
+- job:
+    name: openstack-meta-content-provider
+    parent: openstack-k8s-operators-content-provider
+    description: |
+      A zuul job to build content (rpms, openstack services containers,
+      operators) from opendev and github changes.
+    timeout: 5000
+    run:
+      - ci/playbooks/meta_content_provider/run.yml
+
+- job:
```

Comment:
As description is not inherited from the parent job please define it here as well, maybe yaml anchors work here as well?

Reply:
Done.